data/human/culture republic.txt

```diff
@@ -0,0 +1,125 @@
+			`	Two young adults wander into the aisle just a couple of meters away. They're clearly from abroad judging by their conversation occurring in some local dialect which is neither local nor one you understand. They're cooing over an item they've found but when one flips it over to look at the price tag there is an immediate exchange of looks and expressions that does not require translation. The item ends up back on the shelf with a giggle and a head-shake.`
+			`	Curious you turn over your own mug and see that it is indeed at least five times what you would expect to pay for a mug, even a good one. Still, t is certainly a novelty...`
+			choice
+				`	(Buy the mug.)`
+					goto buy
+				`	(Put it back.)`
+			`	You roll your eyes and put it back. Some experiences really are universal.`
+				decline
+			label buy
+			`	You purchase the mug on an impulse. At least you can ditch your other one with a chip in it.`
+				decline
+
+
+mission "ZZCU Trash Fire Crops"
+	minor
+	source
+		government "Republic"
+		attributes "dirt belt"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`As you move about the spaceport you can't avoid the smell. It's like a trash fire, but nothing more than the smell is in the air.`
+			choice
+				`	(Ignore it.)`
+					goto ignore
+				`	(Ask around.)`
+			`	You ask one of the locals who doesn't look too busy. They sadly inform you that there's been a bumper banana crop, but unfortunately other, wealthier, worlds had a bumper crop at the same time, and there's nowhere to either sell or store a significant excess. Distressed farmers are burning the excess fruit to make fertilizer for the next season, the wind has carried the smell all over town.`
+				decline
+			label ignore
+			`	You leave the mysterious smell to the locals and continue on your business.`
+				decline
+
+
+mission "ZZCU Paradise Cleaner Incident"
+	minor
+	source
+		government "Republic"
+		attributes "paradise"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`Visiting the Paradise spaceports is always an interesting experience for someone from abroad. Nowhere else is class such an ingrained component of daily life. You catch yourself noticing people who seem to be invisible to most of those going about their business, but they never catch your eye. Perhaps it is because of your background, but you notice them even when they are avoiding notice, and so you are able to notice something seconds before it happens.`
```

Comment:
Repetition of "notice".
Also, are the "invisible" people easy to notice for first last or hard? It's really not clear.

Reply:
Removed one notice, but this circumstance is a little bit like that. Hopefully made it more clear.